php_phongo.h

```diff
@@ -35,32 +35,32 @@ extern zend_module_entry mongodb_module_entry;
  * processes do not destroy clients created by other processes (relevant for
  * forking). We avoid using pid_t for Windows compatibility. */
 typedef struct {
-	mongoc_client_t *client;
+	mongoc_client_t* client;
 	int              pid;
 } php_phongo_pclient_t;
 
 ZEND_BEGIN_MODULE_GLOBALS(mongodb)
-	char *debug;
-	FILE *debug_fd;
+	char*             debug;
+	FILE*             debug_fd;
 	bson_mem_vtable_t bsonMemVTable;
-	HashTable pclients;
-	HashTable *subscribers;
+	HashTable         pclients;
+	HashTable*        subscribers;
 ZEND_END_MODULE_GLOBALS(mongodb)
 
 #if PHP_VERSION_ID >= 70000
-# define MONGODB_G(v) ZEND_MODULE_GLOBALS_ACCESSOR(mongodb, v)
-# if defined(ZTS) && defined(COMPILE_DL_MONGODB)
+#define MONGODB_G(v) ZEND_MODULE_GLOBALS_ACCESSOR(mongodb, v)
+#if defined(ZTS) && defined(COMPILE_DL_MONGODB)
 ZEND_TSRMLS_CACHE_EXTERN()
-# endif
-#else
-# ifdef ZTS
-#  define MONGODB_G(v) TSRMG(mongodb_globals_id, zend_mongodb_globals *, v)
-#  define mglo mongodb_globals_id
-# else
-#  define MONGODB_G(v) (mongodb_globals.v)
-#  define mglo mongodb_globals
-# endif
 #endif
+#else /* PHP_VERSION_ID >= 70000 */
```

Comment:
Would it make more sense for this to display the logic for entering the `else` block (e.g `PHP_VERSION_ID < 70000`), or is that uncommon?

Based on what I see below with `ZTS`, I assume it's easier to just repeat the original `#if` condition for matching purposes.